gui/src/routes/home/_l/settings/_l/networks/_l/new.tsx

```diff
@@ -56,6 +56,7 @@ function Content() {
   const onSubmit = async (data: Network) => {
     try {
       setLoading(true);
+      data.dedup_chain_id.dedup_id = 0;
```

Comment:
I know this isn't ideal but this is just temporary so the app remains functional, and we still preventing networks with the same chain_id so this should never fail

Since the `Network` Zod schema is shared between the create and edit form, this meant I had to also update the `NewNetworkParams` struct to have a `dedup_chain_id`, causing this submit to fail if there's no value set to `dedup_id`

I'm now working on a separate branch to see if we can refactor out the need for the NewNetworkParams and go directly to the Network but I haven't finished that yet so for now this was the solution I came up with to move things along